### DISCUSSION TITLE
Neovim OK, but dont let Emacs on the side!

### DISCUSSION THREAD

**Discussion by rakotomandimby** [community/community#58680] [Copilot Conversations]
### Select Topic Area

General

### Body

Hello,

The community ackowledge that there is a NeoVim Copilot client: https://github.com/github/copilot.vim
But, Emacs is also a good choice! For the moment, there is an unofficial client : https://github.com/zerolfx/copilot.el
Please support it, make it officialy supported, and... please, do something for Emacs and Copilot X, please... 

Regards

**Comment by bacchist** [comment]
I second this. Would love emacs support.

**Comment by pbganta** [comment]
+1. Please :-)

**Comment by maestrogerardo** [comment]
I live in Emacs.

Unbelievable that there is no official support for it. 😔

**Comment by brianhuster** [comment]
Copilot now has an official SDK, so should be possible to integrate it to Emacs's LSP client

**Comment by Remillard** [comment]
I'd like to add my support to this.  To be compliant with corporate it needs to be certified with GitHub as a valid safe client.  Please evaluate and add support.  Driving me crazy not to be able to use my preferred environment (and I'm not going to switch to NeoVim -- have been down that road and it was ugly.)
